tardis/util/base.py

```diff
@@ -247,7 +248,7 @@ def create_synpp_yaml(radial1d_mdl, fname, shell_no=0, lines_db=None):
         yaml.dump(yaml_reference, stream=f, explicit_start=True)
 
 
-def intensity_black_body(nu, T):
+def cython_intensity_black_body(nu, T):
```

Comment:
are you working on that?

Reply:
No I'm working on the intensity_black_body function, which is the numba implementation.
The cython_intensity_black_body function is the old function, used to compare the cython performance with numba implementation.

Reply:
I am profiling my code. So far the montecarlo method seems to take a lot of time. I'm searching for more functions which can be optimised.
@wkerzendorf I'll be updating this PR with the profile output soon

Reply:
There was only a difference of 1-2 seconds so I'll optimise other functions.
Using cython
<img width="1013" alt="not-optimal" src="https://user-images.githubusercontent.com/12948588/77286056-dc9fff00-6ca8-11ea-8c81-a28a19da570e.png">

Using numba
<img width="1022" alt="optim" src="https://user-images.githubusercontent.com/12948588/77286026-cd20b600-6ca8-11ea-92f1-de3f7424aed8.png">